public/assets/accordion/accordion.js

```diff
@@ -264,7 +265,8 @@ export default class Accordion
             this.collapse(part, part.collapsed, false)
 
             // Set click event
-            part.handle.addEventListener('click', part.handle.fn = () => {
+            part.handle.addEventListener('click', part.handle.fn = (event) => {
```

Comment:
```suggestion
            part.handle.addEventListener('click', part.handle.fn = () => {
```

That change is not needed anymore

Reply:
Perfect. 👍 Thank You